lib/miq_pglogical/connection_handling.rb

```diff
@@ -15,8 +15,10 @@ def with_connection_error_handling
       end
 
       def pglogical(refresh = false)
-        @pglogical = nil if refresh
-        @pglogical ||= PG::LogicalReplication::Client.new(pg_connection)
+        # TODO: Review if the reasons behind the previous caching / refreshing
```

Comment:
Should we leave the previous code in here as part of the comment?  I know it will be available via the commit that added it, but sometimes `git blame`'ing it after it has been refactored a bunch is a pain.

It would also be a bigger incentive to move this out again.

Reply:
I contemplating making this change bigger and removing the `refresh` completely from here and the method below but decided a more surgical approach would be better.

I'm very open to making this as clear and informative as possible as I will forget these details in a few days anyway 🤣

Reply:
@jrafanie for some sample copy, I was thinking something just like this:

```ruby
# TODO: Review if the reasons behind the previous caching / refreshing
# of the PG::LogicalReplication::Client
#
#    @pglogical = nil if refresh
#    @pglogical ||= PG::LogicalReplication::Client.new(pg_connection)
#
# is still relevant as it caused segfaults with rails 6 when the
# caching was in place.
```

Reply:
I like it, I'll push that change.

Reply:
Fixed.